indico/modules/events/papers/controllers/management.py

```diff
@@ -84,3 +87,27 @@ def _process_PUT(self):
     def _process_DELETE(self):
         set_reviewing_state(self.event_new, request.view_args['reviewing_type'], False)
         return jsonify_data(flash=False, html=_render_paper_dashboard(self.event_new))
+
+
+class RHManageCompetences(RHManagePapersBase):
+    """Manage the competences of the call for papers ACLs"""
+
+    def _process(self):
+        form_class = make_competences_form(self.event_new)
+        user_competences = self.event_new.cfp.user_competences
+        defaults = {'competences_{}'.format(user_id): competences.competences
+                    for user_id, competences in user_competences.iteritems()}
+        form = form_class(obj=FormDefaults(defaults))
+        if form.validate_on_submit():
+            key_prefix = 'competences_'
+            form_data = {int(key[len(key_prefix):]): value for key, value in form.data.iteritems()}
+            for user_id, competences in form_data.iteritems():
+                if user_id in user_competences:
+                    update_competences(user_competences[user_id], competences)
+                elif competences:
+                    user = User.get_one(user_id, is_deleted=False)
```

Comment:
It doesn't make a huge difference here since it only runs on save and you probably won't have *that* many reviewers, but you could query all of them at once:

    users = {u.id: u for u in User.query.filter(User.id.in_(form_data), ~User.is_deleted)}